scheduler/src/cook/kubernetes/api.clj

```diff
@@ -593,6 +604,13 @@
 
     (.setNodeName pod-spec hostname)
     (.setRestartPolicy pod-spec "Never")
+    
+    (.addTolerationsItem pod-spec toleration-for-deletion-candidate-of-autoscaler)
```

Comment:
Some comments on what this achieves would be appreciated.

Reply:
Thanks, will do

Reply:
Added the comment to the `def` for the string constant